src/components/Counter/Counter.vue

```diff
@@ -0,0 +1,73 @@
+<!--
+ - @copyright Copyright (c) 2019 Marco Ambrosini <ma12co@pm.me>
+ -
+ - @author Marco Ambrosini <ma12co@pm.me>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+
+	<!-- counter -->
+	<li :class="{highlighted: isMention}"
+		class="app-navigation-entry-utils-counter">
+		<span v-if="!isMention">{{ item.utils.counter }}</span>
+		<span v-if="isMention">@ {{ item.utils.mentionsCounter }}</span>
```

Comment:
Why not let the dev pass the `@ xx` directly into the counter?
Would be easier and allow more values :)

Reply:
Maybe just add a highlighted prop which is a boolean?

Reply:
yeah, also in talk we dont have a number on the mention (yet).
Maybe we can have a generic list of counters, so the app dev can pick what ever they want.

Reply:
Ok, I'm gonna pass down the displayed text as a slot so you can just do <Counter>wehatever</Counter>